cognite/seismic/protos/v1/seismic_service_messages.proto

```diff
@@ -126,6 +126,8 @@ message CreateSeismicRequest {
         VolumeDef volume_def = 5;  // Define the volume as VolumeDef format
         Geometry geometry = 7;  // Defines the volume as WKT or GeoJson
         bool empty = 10;  // If true, will create a seismic with an empty cutout
+        Seismic2dExtent two_dee_extent = 12;   // only valid if the seismic is 2D
```

Comment:
I'm wondering if we need to support multiple 2d extents here when creating a seismic, one for each requested trace header field? Would they want to be able to define a seismic by both a cdp extent and a shotpoint extent? Or would they just make separate seismics?